x-pack/test/functional/apps/watcher/watcher_test.js

```diff
@@ -35,6 +35,8 @@ export default function({ getService, getPageObjects }) {
       await browser.setWindowSize(1600, 1000);
       // TODO: Remove the retry.try wrapper once https://github.com/elastic/kibana/issues/55985 is resolved
       retry.try(async () => {
+        // Try to give the license time to come through.
+        await PageObjects.common.sleep(2000);
```

Comment:
We need to remove this hard coded sleep. Big no no especially in CI. We would be better off using the `retry.waitFor()` function and figure out a reasonable condition to wait for. We can wait for anything that can return true of false so if we need to wait for a license to be changed or something, we can hit the license API using the ES client.

Reply:
Hey @cuff-links , that sounds reasonable :)